blog/2022-11-09-apache-monitoring.md

```diff
@@ -0,0 +1,144 @@
+---
+slug: apache-monitoring
+title: Apache Monitoring
+description: "Best practices for Apache Server monitoring."
+authors: shyam
+tags: [how-to, monitor, troubleshoot, Apache, web server]
+keywords: [how-to, monitor, troubleshoot, Apache, web server]
+image: https://upload.wikimedia.org/wikipedia/commons/1/10/Apache_HTTP_server_logo_%282019-present%29.svg
+---
+<!--truncate-->
+## What is Apache?
+
+The Apache HTTP Server ([Apache HTTPd](https://httpd.apache.org/)) is one of the most popular open source web servers available. HTTPd was also the first project developed by the Apache Software foundation which now supports hundreds of well known projects including Kafka, Cassandra and Hadoop. 
+
+Netdata has a public loginless [demo space](https://app.netdata.cloud/spaces/netdata-demo) where you can explore different monitoring use-cases. Check out the <a href="https://app.netdata.cloud/spaces/netdata-demo/rooms/apache/overview#chartName=menu_apache">Apache demo room </a>to explore and interact with the charts and metrics described here.
+
+## Monitoring Apache with Netdata
+
+The prerequisites for monitoring Apache with Netdata are to have one or more Apache web server running with [mod_status](https://httpd.apache.org/docs/2.4/mod/mod_status.html) enabled and of course that you have [Netdata installed](https://learn.netdata.cloud/docs/cloud/get-started) on your system. 
+
+The only configuration you need to do is to add the web server's `server-status?auto`. Here is an example:
+
+```yaml
+jobs:
+  - name: local
+    url: http://127.0.0.1/server-status?auto
+```
+You should now see the Apache section on the Overview tab in Netdata Cloud that’s already populated with charts about all the metrics you care about!
+
+For more information please read the Apache collector [documentation](https://learn.netdata.cloud/docs/agent/collectors/go.d.plugin/modules/apache).
+
+## What Apache metrics are important to monitor - and why?
+
+Netdata's Apache summary dashboard helps you get a quick grasp of how your web server is doing in a single glance. The current values for requests, connections, bandwidth and worker utilization are shown here by default and hovering over the time series charts will update the summary charts to reflect the values of these metrics at that point in time.
+
+![image](https://user-images.githubusercontent.com/24860547/200812731-48b84d84-cc29-46ee-a4d2-9293b1cde007.png)
+
+### Requests
+
+The rate of requests received per second is an important metric to monitor. A sudden and substantial increase in the rate of requests is definitely worth digging deeper into. It could be indicative of, for example, a DoS (Denial of Service) attack. Even if the traffic is not malicious in nature you might still need to make changes to ensure your infrastructure is ready to handle the extra load. A significant decrease in the rate of requests could also point to problems that need troubleshooting.
```

Comment:
```suggestion
The rate of requests received per second is an important metric to monitor. A sudden and substantial increase in the rate of requests is definitely worth digging deeper into. It could be indicative of, for example, a [DoS](https://en.wikipedia.org/wiki/Denial-of-service_attack) (Denial of Service) attack. Even if the traffic is not malicious in nature you might still need to make changes to ensure your infrastructure is ready to handle the extra [load](https://blog.netdata.cloud/server-load/#what-is-server-load). A significant decrease in the rate of requests could also point to problems that need troubleshooting.
```